providers-sdk/v1/util/zerologadapter/adapter.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package zerologadapter
```

Comment:
I think this package should be in https://github.com/mondoohq/cnquery/tree/main/logger

Reply:
I put it inside the `providers-sdk/v1` to make it clear that providers can and should use it, Ill move it though.